dashboard/app/controllers/javabuilder_sessions_controller.rb

```diff
@@ -62,7 +62,8 @@ def get_encoded_payload(additional_payload)
     mini_app_type = params[:miniAppType]
     options = options ? options.to_json : '{}'
 
-    issued_at_time = Time.now.to_i
+    # Set the IAT a little in the past to account for time drift between environments
+    issued_at_time = (Time.now - 5.seconds).to_i
```

Comment:
nit: store 5 (or `5.seconds`) as a constant for easy referencing/updating later?

Reply:
I could really see this going either way. In this case I prefer to leave it as is (although that may just be because that's easier than changing it). Trying to come up for a reason for that feeling, here's what I got: I usually use a constant in order to prevent magic numbers, but here, the `.seconds` seems like it does enough to prevent this from being a magic number. Additionally, I think it would be hard to come up with a concise name that would be equally as descriptive as the comment. Since this only gets used in one place right now, I personally like it as the inline form. That being said, I could see it going either way, but in this particular case, the `5.seconds` coupled with the comment feels more descriptive and clean.